sqlglot/dialects/prql.py

```diff
@@ -55,6 +55,16 @@ class Parser(parser.Parser):
             "SORT": lambda self, query: self._parse_order_by(query),
         }
 
+        def _parse_equality(self) -> t.Optional[exp.Expression]:
+            x = self._parse_tokens(self._parse_comparison, self.EQUALITY)
+            # https://prql-lang.org/book/reference/spec/null.html
+            if x and (isinstance(x.expression, exp.Null) or isinstance(x.this, exp.Null)):
```

Comment:
maybe make this two expressions because you have a bunch of if elses on 62 and 63 anyways

if not eq:
   return eq

if isinstance(eq.expresison, exp.Null):
   ...
elif isinstance(eq.this, exp.Null):
  ...
return eq